core/src/test/java/org/apache/iceberg/TestMetricsTruncation.java

```diff
@@ -202,11 +202,20 @@ public void testTruncateStringMax() {
     String test5 = "\uDBFF\uDFFF\uDBFF\uDFFF";
     String test6 = "\uD800\uDFFF\uD800\uDFFF";
     // Increment the previous character
-    String test6_2_expected = "\uD801\uDC00";
+    String test6_1_expected = "\uD801\uDC00";
```

Comment:
I think this should be a typo, "\uD800\uDFFF" is a Unicode surrogate pair, it's length is 1.

Reply:
I'm confused on this one, why is this one changed? Why did the test pass before?